lib/src/checkers/ignore_minimum_tap_area_size.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class IgnoreMinimumTapAreaSize extends InheritedWidget {
```

Comment:
Could you please add a comment on how and when to use this widget? 

I think similar comment should be also added to README so it's visible on pub.dev

Reply:
Done, ok?

Reply:
Yes, thanks!